src/upload.ts

```diff
@@ -508,20 +520,31 @@ const updateVideoInfo = async (videoJSON: VideoToEdit, messageTransport: Message
     await textBoxes[0].focus()
     await page.waitForTimeout(1000)
     await sleep(1000)
+    if (!videoJSON.isKid) {
+        await page.click("tp-yt-paper-radio-button[name='VIDEO_MADE_FOR_KIDS_MFK']").catch()
+    } else {
+        await page.click("tp-yt-paper-radio-button[name='VIDEO_MADE_FOR_KIDS_NOT_MFK']").catch()
+    }
+
+    if (!videoJSON.isRestriction) {
+        await page.click("tp-yt-paper-radio-button[name='VIDEO_AGE_RESTRICTION_SELF']").catch()
+    }
     if (title) {
-        await page.keyboard.down('Control')
-        await page.keyboard.press('A')
-        await page.keyboard.up('Control')
-        await page.keyboard.press('Backspace')
+        // await page.keyboard.down('Control')
+        // await page.keyboard.press('A')
+        // await page.keyboard.up('Control')
+        // await page.keyboard.press('Backspace')
+        await textBoxes[0].evaluate(e => (e as any).__shady_native_textContent = "")
         await textBoxes[0].type(title.substring(0, maxTitleLen))
     }
     // Edit the Description content (if)
     if (description) {
-        await textBoxes[1].focus()
-        await page.keyboard.down('Control')
-        await page.keyboard.press('A')
-        await page.keyboard.up('Control')
-        await page.keyboard.press('Backspace')
+        // await textBoxes[1].focus()
+        // await page.keyboard.down('Control')
+        // await page.keyboard.press('A')
+        // await page.keyboard.up('Control')
+        // await page.keyboard.press('Backspace')
+        await textBoxes[1].evaluate(e => (e as any).__shady_native_textContent = "")
```

Comment:
This code got commented.
Is that on purpose ?
Either code is useful and we keep it, either it's useless and we remove it and explain why.

Reply:
knew it. I forget to explain. That this shortcut unreasonably does not work to clear all the existing text (default video title). that must be replaced with the below 
`await textBoxes[1].evaluate(e => (e as any).__shady_native_textContent = "") `

Reply:
then we should prob delete the other lines rather than commenting them :P